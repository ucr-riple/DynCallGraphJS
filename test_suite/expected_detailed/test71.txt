{
    'global (/Users/madhurimachakraborty/Documents/GitHub/DynCallGraphJS/examples/test71.js:1:1:16:5)': [
        'anon (/Users/madhurimachakraborty/Documents/GitHub/DynCallGraphJS/examples/test71.js:1:2:16:2)'
    ],
    'anon (/Users/madhurimachakraborty/Documents/GitHub/DynCallGraphJS/examples/test71.js:15:3:15:28)': [
        'anon (/Users/madhurimachakraborty/Documents/GitHub/DynCallGraphJS/examples/test71.js:4:17:6:8)'
    ]
}