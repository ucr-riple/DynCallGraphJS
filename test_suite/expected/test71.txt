{ '(test71.js:1:1:16:5)':
   [ '(test71.js:1:2:16:2)' ],
  '(test71.js:15:3:15:28)':
   [ '(test71.js:4:17:6:8)' ] }